### DISCUSSION TITLE
I18N for TinaAdmin

### DISCUSSION THREAD

**Discussion by Perni1984** [tinacms/tinacms#3628] [Ideas]
Hey folks at forestry,

as we're rolling out some projects with Tina in the next weeks and we received some feedback from our test clients that their content editors would love to have a translated backend we were wondering if you would be open to an contribution introducing react-i18n into the admin, thus introducing translation keys. We would be more than happy to provide a first PR for such an implementation, but would like to reach out to you if you have any plans about that already (I don't see anything on the roadmap), if you would be willing to accept such an PR to add that functionality and if so, if you would have any guidance on what we should watch out for, if we're going to tackle i18n for tinaadmin via react-i18n.

**Comment by appinteractive** [comment]
@jeffsee55 any thoughts on that? My clients also have issues with English only. @Perni1984 any workarounds found?
